app/models/spam_address.rb

```diff
@@ -21,6 +21,6 @@ def self.spam?(email_address)
   end
 
   def email=(email)
-    write_attribute(:email, email.downcase)
+    write_attribute(:email, email.downcase.strip)
```

Comment:
Could use the `strip_attributes` gem for this. Would need to call `valid?` in the specs tho.